modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -20,12 +22,24 @@ class ClaimsAndAppealsController < ApplicationController
         end
       end
 
-      before_action(except: %i[index get_claim request_decision]) do
+      before_action(except: %i[index get_claim request_decision upload_document upload_multi_image_document]) do
         authorize :evss, :access?
       end
 
+      before_action(only: %i[upload_document upload_multi_image_document]) do
+        if Flipper.enabled?(:mobile_lighthouse_document_upload)
```

Comment:
Will this flag be for all users or do we want to make it user dependent like we have above?

Reply:
Oh good call, I'll make it per person